File: pinyougou-web-manager/src/main/webapp/js/service/userService.js
app.service("userService",function ($http) {

    //分页查询（不带条件）
    this.queryPage=function (currentPage,itemsPerPage) {
        return $http.get("/user/queryPage.do?pageNum=" + currentPage +
            "&pageSize=" + itemsPerPage)
    }


})

File: pinyougou-interface/src/main/java/cn/itcast/core/service/UserService.java
package cn.itcast.core.service;

import cn.itcast.core.entity.PageResult;
import cn.itcast.core.pojo.user.User;

public interface UserService {
    void sendCode(String phone);

    void add(String smscode, User user);

    User getUserInfomation(String name) throws Exception;

    void saveUserInfomation(User user) throws  Exception;



    PageResult queryPage(int pageNum, int pageSize);
}